docs/src/pages/guides/cli-options.md

```diff
@@ -144,6 +144,14 @@ You can combine this flag with the `--id` flag to repeat custom sequences.
 monika -r 3 -i 1,3,1
 ```
 
+## STUN
+
+By default monika will continuously checking the STUN server in 20 seconds interval. You can specify the number of interval using `-s` or `--stun` flags followed by a number. For example to set the interval to 10 seconds type the command below:
```

Comment:
cosmetic comment: we can add link to explanation what is STUN server.

Reply:
good catch. add this link https://en.wikipedia.org/wiki/STUN

Reply:
also, "You can specify the number of interval using `-s` or `--stun` flags followed by a number in seconds."

Reply:
Also mention, that if the number is less than zero, monika will not check the STUN server.